src/sync.rs

```diff
@@ -3,6 +3,41 @@ use axvcpu::{AccessWidth, AxVCpuExitReason};
 
 use crate::exception_utils::*;
 use crate::TrapFrame;
+use aarch64_cpu::registers::{Readable, ESR_EL2, HCR_EL2, SCTLR_EL1, VTCR_EL2, VTTBR_EL2};
+
+pub fn exception_handle_sync(ctx: &mut TrapFrame) -> AxResult<AxVCpuExitReason> {
```

Comment:
`handle_sync` or `handle_sync_exception`

Also, add annotations for this function, too.

Reply:
or rename it to `sync_exception_handler`, but I prefer the `handle_sync_exception`

anyway, standardize the naming style of these "handle_xxx" functions.

That means you need to rename `data_abort_handler` to `handle_data_abort` if you choose the "handle_xxx" style

Reply:
anyway, standardize the naming style => Yes, I got it.
I want use  `handle_xxxx` style like `handle_sync_exception` `handle_irq_exception` 
And use `xxxx_handler` style in concret handler, like `data_abort_handler`  `reg_handler` or `wfi_handler`